add note about 3rd party Debian repo GPG keys (#1947)
uyuni-project/uyuni-docs: modules/client-configuration/pages/clients-debian.adoc

{debian} clients can require multiple GPG keys to be installed.
====

> When synchronizing third-party {debian} repositories, you will need to import the appropriate GPG key - otherwise the sync will fail. Unlike Red Hat-based repositories, only the metadata is signed - so entering a GPG key url for the software channel is not needed.

Reviewer: ```suggestion
When synchronizing third-party {debian} repositories, you will need to import the appropriate GPG key.
If the GPG key is missing, synchronization will fail.
 
Only the metadata is signed, therefore importing a GPG key for the software channel is not needed.
```

> To see which GPG keys {productname} currently has imported, run the following command:

Reviewer: ```suggestion
To see which GPG keys are already imported to {productname} Server, run the following command:
```

{debian} clients can require multiple GPG keys to be installed.
====

> When synchronizing third-party {debian} repositories, you will need to import the appropriate GPG key.

Reviewer: ```suggestion
When synchronizing third-party {debian} repositories, you will need to import the appropriate GPG key on the server.
```

> If the GPG key is missing, synchronization will fail.

Reviewer 2: This is "on the Server". Not sure if we should make it more clear already here?

> Only the metadata is signed, therefore importing a GPG key for the software channel is not needed.

Reviewer 2: Not sure what this should mean? Do you want to say that the 3rd Party GPG key is not needed on the Client as the metadata are signed only and Uyuni either do not sign them or with an own GPG key which will be trusted on the clients automatically?

Author: Hi @mcalmer,
sorry for the late reply!

The GPG key is missing on the server side (so on Uyuni or SUSE Manager) as otherwise the download will fail. The packages won't be re-sigend by Uyuni.

Author: I tried to rephrase the context in commit dcf3b8f8b283fd1930555dc133bde21fc7f26619 - does this help?
